BaseFilter/sections/antiadblock.txt

```diff
@@ -1092,6 +1092,8 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 !
 ! NOTE: Regular rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/177441
+@@||celebfapper.com/js/adManager.js

```

Comment:
Please add as less specific solution (not sure if they switch to another request, but scriptlets are good alternative)
`celebfapper.com/#%#//scriptlet('prevent-setTimeout', 'adblocktag')`

Reply:
And please move the scriptlet above the exception rule